screens/single-product/single.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/f9a7380946.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://unfold-ui.netlify.app/css/components.css">
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="../cart/cart.css">
+    <link rel="stylesheet" href="./single.css">
+    <title>YAMAHA</title>
+</head>
+
+<body>
+    <div class="navigation-container">
+        <div class="left-nav">
+            <h3 class="nav-brand"><a href="../../index.html"><i class="fas fa-guitar"></i>Upbeat</a></h3>
+        </div>
+
+        <!-- animation on search -->
+        <div class="search-box">
+            <button class="btn-search"><i class="fas fa-search"></i></button>
+            <input type="text" class="input-search" placeholder="What are you Looking for?">
```

Comment:
make this visible in UI.

Reply:
it will be visible when we click on the search icon